stablehlo/dialect/TypeInference.h

```diff
@@ -129,6 +129,11 @@ LogicalResult inferDynamicUpdateSliceOp(
     ValueRange startIndices,
     SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes);
 
+LogicalResult inferFftOp(
+    Optional<Location> location, Value operand, bool isRfft, bool isIrfft,
```

Comment:
How about `isFftTypeRfft` and `isFftTypeIrfft`? That would be consistent with `isTransposeAInvalid` and `isRngDistributionUniform` in that it would also call out the name of the attribute, not just its value.

Reply:
Yeah I hesitated on this but it seems better to be aligned. Updated.